scripts/validate-router-cutover.sh

```
@@ -0,0 +1,83 @@
+fi
+
+# 5. Critical Host Reachability (Reserved IPs)
+echo "Checking critical host reachability (reserved IPs):"
+for host in "${!CRITICAL_HOSTS[@]}"; do
+  expected_ip="${CRITICAL_HOSTS[$host]}"
+  echo -n "  - $host ($expected_ip)... "
+  if ping -c 1 -W 1 "$expected_ip" >/dev/null 2>&1; then
+    echo "OK"
+  else
+    # Try to find current IP if wrong
+    actual_ip=$(ssh "$ROUTER_IP" "/run/wrappers/bin/sudo grep -i \"leased IP address .* to .*$host\" /var/lib/technitium-dns-server/logs/\$(date +%Y-%m-%d).log | tail -n 1 | grep -oE 'leased IP address \[[0-9.]+' | grep -oE '[0-9.]+' | head -n 1" || echo "unknown")
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)